content/en/cloud_cost_management/multisource_querying/_index.md

```diff
@@ -109,6 +107,9 @@ The `all.cost` metric has [Container costs allocated][13] for AWS, Azure, and Go
 
 <div class="alert alert-warning">If your organization tags with any of these FOCUS tags, Datadog recommends updating your tag key on the underlying infrastructure so that tag values do not overlap with FOCUS tag values in Cloud Cost Management.</div>
 
+## Currency conversion
+Cloud Cost Management retrieves the billing currency from each cloud provider's bill. When processing multisource costs from multiple providers with different currencies, each provider’s cost charges are converted to USD. This conversion is performed using the average monthly exchange rate, which is fetched and updated daily from the NetSuite API. This ensures that Cloud Cost Management can conssitently and accurately represent all cost data, regardless of its original currency.
```

Comment:
I would not mention `NetSuite API`. That is just an implementation detail about an internal tool Datadog uses and not something that would make sense to customers.